### PR TITLE
Add warning about data_lock_waits table won't record optimistic transaction blocked by pessimistic transaction

### PR DESCRIPTION
**Pull request by longfangsong**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add warning about `data_lock_waits` table won't record optimistic transaction blocked by pessimistic transaction (since it is frequently asked).

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-data-lock-waits.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ DESC data_lock_waits;
 > **警告：**
 >
 > * 仅拥有 [PROCESS](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_process) 权限的用户可以查询该表。
+> * 目前该表仅能记录**悲观锁等锁**的情况，如果乐观事务被悲观锁阻塞，这一表中不会显示相应等锁信息。
```
Comment (MyonKeminta) [inline information-schema/information-schema-data-lock-waits.md:42]: Consider also mentioning that autocommit transactions are optimistic

**Review by MyonKeminta — COMMENTED** [review]
Rest LGTM

**Review by TomShawn — APPROVED** [review]

**Review by MyonKeminta — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: information-schema/information-schema-data-lock-waits.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ DESC data_lock_waits;
 > **警告：**
 >
 > * 仅拥有 [PROCESS](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_process) 权限的用户可以查询该表。
+> * 目前该表仅能记录**悲观锁等锁**的情况，如果乐观事务（如 autocommit 的事务）被悲观锁阻塞，这一表中不会显示相应等锁信息。
```
Comment (qiancai) [inline information-schema/information-schema-data-lock-waits.md:42]:
```suggestion
> * 目前该表仅能记录**悲观锁等锁**的情况，如果乐观事务（如 autocommit 的事务）被悲观锁阻塞，这一表中不会显示相应等锁信息。
```

Comment (qiancai) [inline information-schema/information-schema-data-lock-waits.md]:
```suggestion
> * 目前该表仅能记录**悲观锁等锁**的情况，如果乐观事务（如 autocommit 的事务）被悲观锁阻塞，这一表中不会显示相应等锁信息。
```

**Pull request merged**